orca-sql/src/main/resources/db/changelog/20200221-partition-name-table.yml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-partition-name-table
+      author: mvulfson
+      changes:
+        - createTable:
+            tableName: partition_name
+            columns:
+              - column:
+                  name: id
+                  type: int
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: true
```

Comment:
Why would `name` be nullable?

Reply:
yeah, i guess it shouldn't be since we won't insert unless we have a partition name, i will change it

Reply:
actually, i think it makes it easier to fix up in case you actually want to go back to a `null` partition name